kafka-rest/pom.xml

```diff
@@ -93,7 +93,10 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>io.github.resilience4j</groupId>
```

Comment:
Do we need to do any open source clearance or project management type stuff around adding a new dependency?

Reply:
This library uses Apache 2 license. I don't think we need to do any extra work.